docs/tutor.rst

```diff
@@ -100,13 +100,30 @@ Contributing to Tutor
 
 Third-party contributions to Tutor and its plugins are more than welcome! Just make sure to follow these guidelines:
 
-- Outside of obvious bugs, contributions should be discussed first in the `official Open edX forum <https://discuss.openedx.org>`__.
-- Once we agree on a high-level solution, you should open a pull request on the `Tutor repository <https://github.com/overhangio/tutor/pulls>`__ or the corresponding plugin.
-- Make sure that all tests pass by running ``make test`` (see above).
-- If your PR is in the Tutor core repository, add a changelog entry by running ``make changelog-entry``. Edit the new file and follow the formatting instructions that it contains.
-- Write a good Git commit title and message: explain why you are making this change, what problem you are solving and which solution you adopted. Link to the relevant conversation topics in the forums and describe your use case. We *love* long, verbose descriptions :) As for the title, `conventional commits <https://www.conventionalcommits.org>`__ are preferred. Check the repo history!
+- **General Discussion**: For anything beyond obvious bugs, initiate a discussion in the `official Open edX forum <https://discuss.openedx.org>`__. This helps in agreeing on a high-level solution.
+- **Pull Requests**: Open a pull request on the `Tutor repository <https://github.com/overhangio/tutor/pulls>`__ for changes to the Tutor core or the corresponding plugin for plugin-specific changes.
+- **Running Tests**: Ensure that all tests pass by running ``make test``. This applies to both the Tutor core and plugin contributions.
+- **Changelog Entry**: If your PR is in the Tutor core repository or affects plugin users, add a changelog entry by running ``make changelog-entry``. Edit the new file as per the formatting instructions.
+- **Code Formatting**: Format your code, if necessary, using ``make format``.
```

Comment:
This should be moved inside the "running tests" section. Something along the lines of: "if formatting tests fail, make sure to run `make format`.